### ISSUE TITLE
Output file order is reversed

### ISSUE THREAD

**Issue by jonathanpdx** [cadithealth/iniherit#4]
When using iniherit from the command line, I'm finding the output file has the opposite order of the input file. Within each section the properties are ordered properly, but the first section (appears at the top of the input file) becomes the last section (appears at the bottom of the output file).

Any chance you can fix this?

Thanks!

**Comment by grabner** [comment]
a quick test seems to indicate that non-inherited sections (i.e. sections that were in the base file) stay in the same order, but inherited sections (coming from other files) do change their order. can you confirm that is what you are seeing?

**Comment by jonathanpdx** [comment]
That's not what I'm seeing. It looks to me like the output is always reversed. For instance, given a base and override files:

```
# mybase.ini
[section1]
section1var = orig1

[section2]
section2var = orig2

[section3]
section3var = orig3

[section4]
section4var = orig4

# override.ini

[DEFAULT]
# the following will cause both "base.ini" and "override.ini" to be
# inherited. the "?" indicates that "override.ini" will be ignored
# if not found; otherwise an error would occur.
%inherit = mybase.ini

[section3]
section3var = over3
```
Then running
```
$ iniherit override.ini output.ini ; cat output.ini
[section4]
section4var = orig4

[section3]
section3var = over3

[section2]
section2var = orig2

[section1]
section1var = orig1
```

I've tried a few variations and it seems pretty consistent: The output file is always in the reverse order of the base file.

**Comment by grabner** [comment]
although i was able to reproduce that, when playing around with the section names a bit, i discovered that the order is somewhat unpredictable. unfortunately, that combined with the fact that iniherit just uses the standard ConfigParser to generate the output which (IIRC) doesn't really give much control on ordering, tells me that this may be more difficult to make happen than it would appear.

either way, i'll take a look soon (perhaps tonight) and let you know.

**Comment by jonathanpdx** [comment]
Perhaps this is helpful?

http://stackoverflow.com/questions/1134071/keep-configparser-output-files-sorted

**Comment by grabner** [comment]
that definitely seems to indicate that it can be solved in python 2.7+/3.1+!

**Comment by jonathanpdx** [comment]
Yup. This particular user is running 2.7, if that's at all relevant. :)

**Comment by grabner** [comment]
fixed in 0.3.3.

thanks for the bug report!
